src/components/Header/components/MobileNav.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Box,
+  chakra,
+  Flex,
+  Spacer,
+  useMultiStyleConfig,
+} from "@chakra-ui/react";
+import React from "react";
+
+import Link from "../../Link/Link";
+import List from "../../List/List";
+import Logo from "../../Logo/Logo";
+import SimpleGrid from "../../Grid/SimpleGrid";
+
+/**
+ * The Header navigation for the mobile view.
+ */
+const MobileNav = chakra(() => {
+  const styles = useMultiStyleConfig("HeaderMobileNav", {});
+
+  return (
+    <Box __css={styles}>
+      <Flex>
+        <Box>
+          <Logo
+            name="nyplLionWhite"
+            size="xxsmall"
+            marginTop="15px"
+            marginLeft="15px"
+            decorative={false}
+            title="NYPL Header Logo"
+          />
+        </Box>
+        <Spacer />
+        <nav aria-label="Header mobile links">
+          <List
+            id="header-mobile-nav"
+            listItems={[
+              <Link href="#" key="booksMusicMoviesLink">
+                Books/Music/Movies
+              </Link>,
+              <Link href="#" key="researchLink">
+                Research
+              </Link>,
+              <Link href="#" key="educationLink">
+                Education
+              </Link>,
+              <Link href="#" key="eventsLink">
+                Events
+              </Link>,
+              <Link href="#" key="connectLink">
+                Connect
+              </Link>,
+              <Link href="#" key="giveLink">
+                Give
+              </Link>,
+              <Link href="#" key="getHelpLink">
+                Get Help
+              </Link>,
+            ]}
+            noStyling
+            type="ul"
+            __css={styles.sideNav}
+          />
+        </nav>
+      </Flex>
+      <SimpleGrid gap="0" data-testid="bottomLinks" __css={styles.bottomLinks}>
```

Comment:
By adding `__css={styles.bottomLinks}`, the `SimpleGrid` component somehow lost its `display: "grid"` style. Have you seen this before? I had to manually add that rule in "bottomLinks".

Reply:
No, I have not seen that before.  I am surious what else is in `bottomLink`.  I will check when I review the files.

Reply:
I was not able to pinpoint why this is happening.  I looked at nesting, props on child elements, etc., but I could not find anything specifically tied to this issue.  Despite not finding anything, I feel it's related to nesting and the mobile sizing.

Let's move forward with what you have done and keep our eyes open for possible causes.